applications/Chat/evaluate/evaluator.py

```diff
@@ -0,0 +1,68 @@
+import metrics as metrics
+import pandas as pd
+
+
+class Evaluator(object):
+    """
+        A class named Evaluator includes GPT-3.5/GPT-4 evaluation
+        and automatic evaluation
+
+    """
+
+    def __init__(self, params: dict) -> None:
+        self.params = params
+        self.stats = dict()
+
+    def battle(self, answers1: dict, answers2: dict) -> None:
+        """
+        Comparison between two models using GPT-4 as the reviewer.
+        """
+        pass
+
+    def evaluate(self, answers: dict, targets: dict) -> None:
```

Comment:
The `answers` and `targets` should be `list` instead? `jload` fuction can return `dict` and `list` type.

If we use `predict_folder`, targets will be a dict like:
```python
{
    "name1": answers_list1,
    "name2": answers_list2,
    ......
}
```